packages/Radio/src/Radio.styles.js

```diff
@@ -9,12 +9,29 @@ const getLabelLeftPadding = (radioSize, hasLabel) =>
 
 const smallRadioSize = tokens.radio.sizeSm;
 const mediumRadioSize = tokens.radio.sizeMd;
-const largeRadioSize = tokens.radio.sizeLg;
 
 const getHalfSizeCss = sizeCss => `${toInt(sizeCss) / 2}px`;
 const smallRadioHalfSize = getHalfSizeCss(smallRadioSize);
 const mediumRadioHalfSize = getHalfSizeCss(mediumRadioSize);
-const largeRadioHalfSize = getHalfSizeCss(largeRadioSize);
+
+const generalRadioStyle = {
+  radioStyles: {
+    height: mediumRadioSize,
+    width: mediumRadioSize,
+    borderRadius: mediumRadioHalfSize,
+  },
+  radioIconBackgroundStyles: {
+    borderRadius: "6px",
+    height: "10px",
+    top: "5px",
+    width: "10px",
+  },
+  radioIconStyles: {
+    fontSize: `${fontSizeValue(-2)}px`,
```

Comment:
fyi `${stylers.fontSize(-2)};` is the same as `fontSize: `${fontSizeValue(-2)}px`,`